test/hotspot/jtreg/vmTestbase/nsk/jdi/EventRequest/hashCode/hashcode001.java

```diff
@@ -118,7 +118,7 @@ private static void execTest() {
             switch (i) {
 
                 case 0:
-                       ThreadReference thread = debuggee.threadByNameOrThrow(methodName);
+                       ThreadReference thread = debuggee.mainThread();
```

Comment:
This one is not so obvious until you look earlier in the file and see:

```    private final static String methodName = "main";```

So `methodName` is "main". Probably should have always explicitly used "main" here instead, but with this change it's gone anyway.